adapters/adform/adform.go

```diff
@@ -61,6 +62,9 @@ type adformAdUnit struct {
 	PriceType   string      `json:"priceType,omitempty"`
 	KeyValues   string      `json:"mkv,omitempty"`
 	KeyWords    string      `json:"mkw,omitempty"`
+	CDims       string      `json:"cdims,omitempty"`
+	MinPrice    float64     `json:"minp,omitempty"`
+	Url         string      `json:"url,omitempty"`
```

Comment:
In order to validate the params your adapter uses, these new parameters must be declared in `openrtb_ext/imp_adform.go`. Even if you end up unmarshaling to an `adformAdUnit` struct as `adapters/adform/adform.go` does in lines 178 and 435, the core prebid-server performs its validation of the `Imp` extensions found in `openrtb_ext/imp_adform.go`.

Reply:
Somehow missed that. Thanks for the note.